rpc/gnmi/mixed-schema.md

```diff
@@ -108,50 +109,187 @@ if all transactions succeed should the `SetResponse` indicate success. If any
 of the transactions fail, the contents of all origins MUST be rolled back, and
 an error status returned upon responding to the `Set` RPC.
 
-## Example
+## Special considerations for a setRequest with openconfig and cli origins
+
+### Use cases
+The operational use cases for a setRequest `replace` operation containing both OC and CLI origins include:
+
+1. The ability to support incremental transition from CLI based configuration to OpenConfig configuration.
+2. A device may implement configuration items which are only defined in CLI while other configuration items are supported with OpenConfig.
+3. The desire to use a full configuration replacement strategy.
+
+To support these use cases a special meaning to a setRequest with replace origin `cli` and replace origin `openconfig` is defined.
+
+### SetRequest behavior
+
+A `SetRequest` message using the `replace` operation for both `cli` and `openconfig` origins SHOULD be performed as follows:
 
-### Example: OpenConfig and CLI Data
+1. `origin` `cli` SHOULD appear first.  
```

Comment:
what does "appear" mean here?  don't think it means that the client must provide it first, as your example provided does not do that. Perhaps a better term would be "applied"?

Also, the word should/SHOULD is used in the steps where I think the intention is to say MUST.

Reply:
@sebsapa I think it means exactly that - the client should form a Set request in such a way that Updates with origin=cli appear first in the list of Updates.

In the provided example I see the first replace message is specified with `origin=cli` and the 2nd one with `openconfig`